app/js/lib/foxbox.js

```diff
@@ -327,6 +326,25 @@ export default class Foxbox extends Service {
     });
   }
 
+  performServiceCommand(id, command, method = 'GET', state = undefined) {
```

Comment:
OK, let's leave it for now as a workaround until "`frontend adapters`" land. They'll be part of the presentation I'll give during the workweek, but let's make sure we specify them and come with a plan regarding the implementation.
I do think that the lib should be kept in sync with the box so that we don't need to extend it. I believe the 3rd party dev should completely ignore the underlying REST API and only work with a user friendly, high level API.
Exposing such a low-level method may harm retrocompatibility in the future and block us from making any changes to the REST API implemented in the box.
So in the meantime, please add a big comment saying something like `Do not use! Temporary fix will be deprecated very soon` and show this warning in a `console.warn()` too.